### PR TITLE
Allow local running outside of docker for psd-web/worker

### PR DESCRIPTION
**Pull request by doctorpod**
## Checklist:

- [ ] Automated checks are passing locally.
- [ ] CHANGELOG updated if change is worth telling users about.
### General testing
- [ ] Test without javascript
- [ ] Test on small screen
### Accessibility testing
- [ ] Works keyboard only
- [ ] Tested with one screen reader
- [ ] Zoom page to 400% - content still visible
- [ ] Disable css - does content make sense and appear in a logical order?
- [ ] Passes automated checker (automated in build or manual)

### CODE REVIEW

File: psd-web/bin/run_local [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+export RUN_LOCAL=true
+
+echo
+echo 'Installing chromdriver if required. This is needed to run system tests...'
+echo '======================================================='
+if which chromedriver
+then
+  echo 'Already installed'
+else
+  brew update && brew install imagemagick
```
Comment (baphled) [inline psd-web/bin/run_local]: We should be careful with installing apps that are dependant of a OS 🤔 :)

File: psd-web/bin/run_local [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+echo '...done'
+
+echo
+echo 'Ensure all node packages are installed...'
+yarn install
+echo '...done'
+
+echo
+echo 'Starting backgroud worker...'
+bin/sidekiq -C config/sidekiq.yml &
```
Comment (baphled) [inline psd-web/bin/run_local]: Procfile ftw :)

File: psd-web/config/database.yml [hunk 3]
```diff
@@ -1,7 +1,7 @@
 default: &default
   adapter: postgresql
   encoding: unicode
-  host: db
+  host: <%= ENV['RUN_LOCAL'] == 'true' ? 'localhost' : 'db' %>
```
Comment (baphled) [inline psd-web/config/database.yml]: We could just use the `.env` to populate this value.

So `.env.development` has `DB_HOST=localhost`

File: psd-web/docker-compose-backing-services.yml [hunk 4]
```diff
@@ -0,0 +1,55 @@
+version: "3"
+
+services:
+  db:
+    build:
+      context: ../db
+    volumes:
+      - postgres-volume:/var/lib/postgresql/data
+    env_file:
+      - ../.env.keycloak
```
Comment (matthewford) [inline psd-web/docker-compose-backing-services.yml]: is this working for people without adding the DB ports?

Comment (baphled) [reply psd-web/docker-compose-backing-services.yml]: I haven't tested this locally, myself.

**Comment by slorek** [comment]
Closing as not updated for several weeks.

**Pull request closed without merging**